### DISCUSSION TITLE
setup.py install does not build .egg?

### DISCUSSION THREAD

**Discussion by 93578237** [pypa/setuptools#3213] [Q&A] [answered]
After v61 release `python setup.py install` does not build .egg, only .egg-info. Is it correct behavior?

**Comment by abravalheri** [comment, marked as answer]
Hi @q0w, thank you very much for reporting this.
The pipeline for the release of a bugfix is running right now. Would it be possible for you to wait until it is published and try again? I would be very nice to have some feedback if the problem still persists (I tested the fix, but it is always impossible to cover all edge cases).

**Reply by abravalheri** [reply]
(Obs.: The release pipeline in the CI does tend to run very slowly because of all the integration tests, but eventually it should be out 😅)

**Reply by 93578237** [reply]
I've cloned the repo and tested it. It works. Thx!
